bundles/org.openhab.binding.ntp/src/main/java/org/openhab/binding/ntp/internal/handler/NtpHandler.java

```diff
@@ -54,217 +54,183 @@
  * to one of the channels.
  *
  * @author Marcel Verpaalen - Initial contribution OH2 ntp binding
- * @author Thomas.Eichstaedt-Engelen OH1 ntp binding (getTime routine)
+ * @author Thomas.Eichstaedt - Engelen OH1 ntp binding (getTime routine)
```

Comment:
This looks wrong. Thomas's last name is "Eichstaedt-Engelen", so the spaces arounf the dash are wrong.

Reply:
There was a warning on this line but my change was stupid.

Reply:
Fixed